internal/policies/manager_test.go

```diff
@@ -115,7 +120,27 @@ func TestApplyPolicies(t *testing.T) {
 				testutils.MakeReadOnly(t, filepath.Join(fakeRootDir, tc.makeDirReadOnly))
 			}
 
+			// capture log output (set to stderr, but captured when loading logrus)
+			r, w, err := os.Pipe()
+			require.NoError(t, err, "Setup: pipe shouldn’t fail")
+			orig := logrus.StandardLogger().Out
+			logrus.StandardLogger().SetOutput(w)
+
 			err = m.ApplyPolicies(context.Background(), "hostname", true, &pols)
+
+			logrus.StandardLogger().SetOutput(orig)
+			w.Close()
+
+			var out bytes.Buffer
```

Comment:
Do we want to do this before checking if there is an error (expected/unexpected)?